### DISCUSSION TITLE
Make a glyphs sale for Honor Points instead of gold

### DISCUSSION THREAD

**Discussion by Plasmashield** [chromiecraft/chromiecraft#3790] [Suggestions]
I suggest to make glyphs sale for honor points. This will solve the problem with unavailable glyphs and at the same time increase activity in PvP. This will also allow you to withdraw excess honor points, just like gems in Wotlk. And will allow players to sell them on the AH, which will have a more positive impact on the server economy and will be more friendly to new players. I believe that allowing players to earn money by pvp and to regulate glyphs prices themselves at the AH will have a more positive impact on the server economy and the players' interest in PvP than just sale glyphs for gold.

**Comment by balleny** [comment]
I think a lot of ways how the glyphs can get accessible have been posted in discord, including access via honor. Unfortunately, those have been dismissed as it seems. Exchange for gold might be the easiest solution, nevertheless this is also gating access for low level players or new players.

Honor would be a different way to get access, although I would suggest a system slighty different:

An NPC to buy the glyphs, tbc glyphs for e.g. for 4 tokens, wrath glyphs for 6 tokens.
The token can be collected via:
exchange of bg marks (e.g. 3 marks of each bg, or 15 marks of an individual bg)
and token have an 20% or 25% drop chance on the rdf satchels

This way, glyphs would be accessible for all levels and would promote both pve and pvp content (and could be combined for token collection)

**Comment by kerricsson** [comment]
Hi all,

I also support this initiative, because it will contribute to PVP scene and help PVPer play game like they want (grind honor instead of gold) and it will be also sinergy during R14 weap grind (they will have additional source to spend honor that means longer live of PVP BGs).

**Comment by Never6** [comment]
I agree with having an option for glyphs to cost honor for the simple reason that I play multiple specs.
Having to spend 600g just so I can play a different spec doesn't make me want to play multiple specs

**Comment by Amrx2** [comment]
Hi all, just to confirm Glyphs will be purchasable by both gold & honor, thanks for raising this @Plasmashield again. 

Cheers,
Construct
